templates/base/head_navbar.tmpl

```diff
@@ -121,13 +121,13 @@
 			<a href="{{AppSubUrl}}/notifications" class="item tooltip not-mobile gt-mx-0" data-content="{{.locale.Tr "notifications"}}" aria-label="{{.locale.Tr "notifications"}}">
 				<span class="fitted item">
 					{{svg "octicon-bell"}}
-					<span class="notification_count{{if not $notificationUnreadCount}} gt-hidden{{end}}">
+					<span class="notification_count{{if not $notificationUnreadCount}} gt-hidden{{end}}"><!-- Nested because the rendering breaks otherwise-->
 						{{$notificationUnreadCount}}
 					</span>
 				</span>
 			</a>
 
-			<div class="ui dropdown jump item tooltip gt-mx-0" data-content="{{.locale.Tr "create_new"}}">
+			<button class="ui dropdown jump item tooltip gt-mx-0" role="menu" data-content="{{.locale.Tr "create_new"}}">
```

Comment:
It shouldn't be a button. Because there are a lot of `div` inside.

Reply:
What do you mean, `there are a lot of divs inside`?
That's the content of this dropdown menu.
Of course, there will be divs to construct this dropdown.
But they are not a part of the shown content of this button, so I doubt that it matters?

Reply:
It doesn't make sense to put `div` or `a` inside a button.

Reply:
https://stackoverflow.com/questions/12982269/is-it-semantically-incorrect-to-put-a-div-or-span-inside-of-a-button

Everyone would say no ....

Reply:
Well… The funny thing is:
As I said, while they are inside the button, **they are apparently not considered children of the button**:
![image](https://user-images.githubusercontent.com/51889757/221236132-0b9f93dc-57c6-4b30-8455-00a6725fea91.png)
Don't ask me how, but if Firefox supports this, I doubt that other browsers don't…

Reply:
Supported doesn't mean it is correct.

Don't ask me why ..... that's basic common sense.

Reply:
The problem I can see is the following:
As I mentioned in the PR description, a `button` is the only semantically valid element for how our drop-downs behave:
You should be able to focus and click on it, and once that's done, it either displays its children, or it hides them again.
That's not the description of a `div`, but the description of a `button`.
The fact that we seem to perform some black magic at the moment to create our dropdowns rather means we should re-evaluate how to create dropdowns than do the semantically incorrect thing of disguising a `div` as a `button`.

Reply:
`button` is not a semantically valid element. Button shouldn't contain other complex elements.

You could just do a `<div role=...` , I do not understand why `div` is not clear, as long as you have set the `role` for it.

Reply:
And, you do not need to do anything for the `dropdown`. Our `aria.js` framework already does so, you can try it on try.gitea.io


![image](https://user-images.githubusercontent.com/2114189/221241722-8d69f500-f379-4280-ae75-d81a6c0e2fc9.png)